File: python/2_struct_inicializacion.py
from dataclasses import dataclass

@dataclass
class Estudiante:
    nombre: str
    edad: int
    promedio: float

estudiante1 = Estudiante("María López", 20, 4.5)
estudiante2 = Estudiante("Carlos Ramírez", 22, 3.8)
estudiante3 = Estudiante("Ana Gómez", 19, 4.2)

print("Estudiante 1:", estudiante1)
print("Estudiante 2:", estudiante2)
print("Estudiante 3:", estudiante3)

File: javascript/2_struct_inicializacion.js
class Estudiante {
    constructor(nombre, edad, promedio) {
        this.nombre = nombre;
        this.edad = edad;
        this.promedio = promedio;
    }
}

const estudiante1 = new Estudiante("María López", 20, 4.5);
const estudiante2 = new Estudiante("Carlos Ramírez", 22, 3.8);
const estudiante3 = new Estudiante("Ana Gómez", 19, 4.2);

console.log("Estudiante 1:", estudiante1);
console.log("Estudiante 2:", estudiante2);
console.log("Estudiante 3:", estudiante3);